test/functional/core/job_spec.lua

```diff
@@ -875,6 +875,13 @@ describe('jobs', function()
     end)
   end)
 
+  it('jobstop fails silently and returns exit status', function()
+    nvim('command', "let j = jobstart(['cat'], {'rpc': v:true})")
+    neq(0, eval('j'))
+    nvim('command', "let h = jobstop(j)")
+    eq(0, eval('h'))
```

Comment:
i assume you're still working on this? 0=success, but this should test a failed jobstop(). E.g. try stopping the same job twice.